ci/cache.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Some comments about what is going on here. Also, this feels like a place where some high-level abstractions might exist that will cross bsg-repos? Or maybe a template?

Reply:
I'm actually going to use the gitlab ci cache, which seems to be able to handle all these operations.  So this file will just go away :)